tests/Unit/Common/EMSLinkTest.php

```diff
@@ -10,17 +10,33 @@ class EMSLinkTest extends TestCase
     public function testFromTextWithEmsObjectTypeAndOuuid()
     {
         $link = EMSLink::fromText('ems://object:page:AWTLzKLc8K-kdP4iJ3rt');
-
         static::assertSame('AWTLzKLc8K-kdP4iJ3rt', $link->getOuuid());
         static::assertSame('page', $link->getContentType());
         static::assertSame('object', $link->getLinkType());
         static::assertSame('ems://object:page:AWTLzKLc8K-kdP4iJ3rt', (string) $link);
     }
 
+    public function testGetOuuid()
+    {
+        $link = EMSLink::fromText('ems://object:page:AWTLzKLc8K-kdP4iJ3rt');
+        static::assertSame('AWTLzKLc8K-kdP4iJ3rt', $link->getOuuid());
+    }
+
+    public function testGetcontentType()
+    {
+        $link = EMSLink::fromText('ems://object:page:AWTLzKLc8K-kdP4iJ3rt');
+        static::assertSame('page', $link->getContentType());
+    }
+
+    public function testGetLinkType()
+    {
+        $link = EMSLink::fromText('ems://object:page:AWTLzKLc8K-kdP4iJ3rt');
+        static::assertSame('object', $link->getLinkType());
+    }
+
```

Comment:
We do not test Setters/Getters, because they are tested implicitly by the other two test.
As we are testing "::fromText" (short and long version) we do not need separate getter tests.